README.md

```diff
@@ -1,2 +1,65 @@
 # Social-Distance-Checker
 Open Projects 2021
+
+
+![socialdistancing_mask_detection](https://user-images.githubusercontent.com/88554453/128603773-dceafea5-7f4f-46a8-8303-98dc23235368.jpg)
+
+# Abstract 
+
+The social distance checker is an opencv project in which we monitor the distances between the people and if the distance comes out to be less than the threshold value (calibrated six feet in terms of pixels) then we indicate the violation by surrounding them with a red bounding box and with the help of AI/ML/pretrained files , if the person does not wear a mask then his face gets surrounded by a red bounding box.
```

Comment:
Justify the entire text using the `<p>` paragraph tag and `justify` attribute. Make sure you justify all the paragraphs separately. For eg:
`<p align="justify">`
  `<h2>Abstract</h2>`
  `<p>A Rube Goldberg Machine is .........................</p>`
`</p>`

Refer this README for the same: https://raw.githubusercontent.com/marsiitr/Rube-Goldberg-Machine/master/README.md